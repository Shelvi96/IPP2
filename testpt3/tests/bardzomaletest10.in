NEW ID

037>12
759390;4;3>::
3920060;8>3;;:55902
7>;68:9
0017>481:00:472
5717;756>84
;6;;6>77
70>74
;;4:8>;367
069428:608>7306
;1754:>2746
@53231;6194163813296780;7562
@044987096::7245507078
@64;:;44568:;5::5;8897:5146
@4583;0294;595178:5:
@4:4662;;:9;76;:
@35;745516429205:3
@89;08:1;98
@7:549
@82
@2124:883;;0703;05
