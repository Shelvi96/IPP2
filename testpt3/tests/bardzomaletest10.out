138946783587557
19190925
1130105566413
805255
21
5357
0
0
0
5000
